index.html

```diff
@@ -163,7 +163,39 @@ <h4 class="poppins-regular">Skills<img src="https://img.icons8.com/small/16/0000
       <hr/>
 
   </section>
+  <footer>
+    <div class="footer-container">
+    <h2 class="poppins contact">Contact me</h2>
+    <p class="poppins-regular">If you have an application you are interested in developing, 
+      a feature that you need built or a project that needs coding. 
+      I’d love to help with it </p>
+
+    <div class="form-container">
+    <form action="https://formspree.io/f/xzbwropn" method="POST">
+
+      <ul class="form-list">
+
+        <li class="form-field">
+          <label for="name"></label>
+          <input class="text-field" type="text" name="name" id="name" max="30" placeholder="name" required>
```

Comment:
- [ ] Kindly note that the correct attribute is `maxlength="30"`. Please replace `max="30"`.